.github/workflows/pull-request-teardown.yml

```diff
@@ -4,6 +4,15 @@ on:
     types: [closed]
 
 jobs:
+  github-env:
+    runs-on: ubuntu-latest
+    steps:
+    - name: get environment
+      run: env
+    - name: Dump GitHub context
+      env:
+        GITHUB_CONTEXT: ${{ toJson(github) }}
+      run: echo "$GITHUB_CONTEXT"
```

Comment:
if you want to get $GITHUB_CONTEXT variable written in the logs, instead of using `echo` you should use something like `echo "::warning file=app.js,line=1,col=5::Start configuration, github context: $GITHUB_CONTEXT"`

Reply:
sorry, should have labeled that as draft - my bad